core/Columns/Dimension.php

```diff
@@ -456,7 +456,7 @@ public function formatValue($value, $idSite, Formatter $formatter)
             case Dimension::TYPE_DURATION_S:
                 return $formatter->getPrettyTimeFromSeconds($value, $displayAsSentence = false);
             case Dimension::TYPE_DURATION_MS:
-                return $formatter->getPrettyTimeFromSeconds($value, $displayAsSentence = true);
+                return $formatter->getPrettyTimeFromSeconds($value/1000, $displayAsSentence = true);
```

Comment:
@sgiehl might be good to use same logic as here? https://github.com/matomo-org/matomo/blob/3.14.1-b1/core/Plugin/ArchivedMetric.php#L151-L155

Reply:
updated it